common/event_modifiers/31_pony.txt

```diff
@@ -17,6 +17,19 @@ running_of_the_leaves = {
 	production_efficiency = 0.10
 }
 
+magic_assimilation = {
+	culture_conversion_cost = -0.8
+	missionaries = 2
+	global_missionary_strength = 0.8
+}
```

Comment:
I'm being silly. Erase if you will, but I wanted a lazy game and good incentives to invade the European continent.
Unfortunately, the europeans are quicker that the equines and starts invading North America quicker than you get to block off all the coast lines. Oh well